play.pokemonshowdown.com/js/client-mainmenu.js

```diff
@@ -949,7 +949,7 @@
 			if (bestOf && bestOfValue) {
 				var hasCustomRules = format.includes('@@@');
 				format += hasCustomRules ? ', ' : '@@@';
-				format += 'Best of = ' + bestOfValue;
+				format += '!!Best of = ' + bestOfValue;
```

Comment:
Should there be logic here to only add `!!` if the best of rule already exists?

Reply:
> Should there be logic here to only add `!!` if the best of rule already exists?

I don't see why it would make a difference personally.

Reply:
![image](https://github.com/smogon/pokemon-showdown-client/assets/11083252/4911cef3-c25d-4d9e-be22-c5e33ff35b71)
because of this

Reply:
I didn't know that was a thing, my bad.

Reply:
I did try to make it conditional but I couldn't really figure out how to make sure it would cover all possible syntaxes for the best-of command, then I thought it wouldn't be needed so I let if be.